src/QsCompiler/Tests.Compiler/SandboxInputSchemaTests.fs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.QsCompiler.Testing
+
+open System.Collections.Generic
+open System.IO
+open System.Text
+open Bond
+open Xunit
+open Xunit.Abstractions
+open Microsoft.Quantum.QsCompiler.BondSchemas.EntryPoint
+open Microsoft.Quantum.QsCompiler.BondSchemas.SandboxInput
+
+type SandboxInputSchemaTests(output: ITestOutputHelper) =
+
+    let createEntryPointOperation() =
+        let expectedEntryPointOperation = new EntryPointOperation()
+        let argument = new Argument()
+        argument.Name <- "name"
+        argument.Position <- 44
```

Comment:
Nit: consider using more explicit names like "Test Argument" and a valid position in this case (which would be 0 since there is only 1 argument). Same for the name of the entry-point operation.